### PR TITLE
Ticket/739 pdq styles

### PR DESCRIPTION
**Pull request by alifr**
Adding PDQ Cancer Information Summary page layout templates, JavaScript library and styles

### CODE REVIEW

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content_page_macros.twig [hunk 1]
```diff
@@ -44,3 +44,36 @@
     {% endif %}
   {% endif %}
 {% endmacro %}
+
+
+{% macro displayPublicUse() %}
```
Comment (alifr) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content_page_macros.twig]: These are going to need translations

Comment (bryanpizzillo) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content_page_macros.twig]: Actually, these need to be removed. Public use is being handled a different way.

Comment (alifr) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content_page_macros.twig]: There's also blocks for Blog Comment Policy, Syndication Use and Press Contacts. Are these all being handled in a custom manner?

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/pdq/node--pdq-cancer-information-summary.html.twig [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+<article>
+{# {{kint(_context)}} #}
+    <div class="resize-content">
+        <h1>{{- node.label -}}</h1>
+        {{ drupal_block('page_options') }}
+        <div id="cgvBody">
+            <div class="summary-sections">
+                {% if content.field_summary_sections|length > 0 %}
+                <nav class="on-this-page" role="navigation">
+                    <h6>{% trans %}On This Page{% endtrans %}</h6>
```
Comment (alifr) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/pdq/node--pdq-cancer-information-summary.html.twig]: Added `trans` twig block, but it does not seem to work. No translation on Spanish pages. Seeing the same for Article Pages where I grabbed the code from.

**Comment by bryanpizzillo** [comment]
This is on http://ncigovcdode36.prod.acquia-sites.com

**Pull request merged**